geometry/meshop.cpp

```diff
@@ -252,6 +254,122 @@ void saveAsPly( const Mesh &mesh, const boost::filesystem::path &filepath){
     }
 }
 
+void saveAsPlyWithFeatures(const boost::filesystem::path& filepath,
```

Comment:
Not sure if this function should stay here as I moved the loading to an internal module. @matyas-hollmann

Reply:
The load/save functionality should be closer together. One option is to have a general PlyWriter and PlyReader here or if not possible have some single-use implementation internally where needed.

Reply:
I moved the function to an internal module. Implementing a general PlyWriter would be neat, but is out of scope of the current MR.